bin/run.sh

```diff
@@ -8,15 +8,10 @@ fi
 
 # Ensure we fail the job if any steps fail
 set -e -o pipefail
-#. $(dirname "$0")/env.sh
 
 # Run the compose file for blackbox testing
 export network=$(docker network ls | awk '{print $2}' | grep edgex-network)
-docker-compose -f ../docker-compose-test-tools.yml up -d app-service-configurable
-
-option="${1}"
-
-echo " ${option}"
+docker-compose -f "$(dirname "$0")"/../docker-compose-test-tools.yml up -d app-service-configurable
```

Comment:
The `$(dirname "$0")` is `bin`, which causes ERROR here. The `docker-compose-test-tools.yml` is under the parent directory.

Reply:
What should be happening is `bin/run.sh` and `cd bin && ./run.sh` should now both work. In the former case `$(dirname $0)` will be `bin` so `bin/../docker-....yml` should be found. In the latter case, `$(dirname $0)` will be `.` so `./../docker-....yml` should be found. I'm seeing this in my testing.

How do I look at the failure in the Jenkins build?

Reply:
The issue was not line 14 but line 147. The `updateMaxResultCount.sh` script didn't have an exit statement and the `set -e` on line 12 was triggered. This is corrected now.

Reply:
When executing `bin/run.sh -cd`, I got the following ERROR:
<img width="806" alt="image" src="https://user-images.githubusercontent.com/55377716/74898935-54db8200-53d6-11ea-9fd7-81b52cc74dcd.png">
In this case, `$(dirname $0)` should be added to all the test scripts.

`cd bin && ./run.sh -cd`works fine.

Reply:
Ah. I see what you're seeing now. Opening another issue to track.